ichnaea/data/station.py

```diff
@@ -400,16 +400,22 @@ def submit_key(self):
         return values
 
 
+def is_retryable(exception):
+    """Return True if the exception is retryable."""
```

Comment:
This should talk about what circumstance is covered. This isn't "is any exception retryable". This is more of a "is a SQLAlchemy exception retryable".

Maybe change the function name to `is_sqlalchemy_exception_retryable`? That's a bit of a mouthful. It's only used in one place, so maybe fold it into `should_giveup`?